cicd/3-app/javabuilder/template.yml.erb

```diff
@@ -973,6 +974,21 @@ Resources:
         AttributeName: <%=TIME_TO_LIVE_ATTRIBUTE_NAME%>
         Enabled: true
 
+  UnhealthyContainersTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: !Sub "${SubDomainName}_unhealthy_containers"
+        KeySchema:
+          - AttributeName: <%=CONTAINER_ID_AND_TRIGGER_COMPOSITE_ATTRIBUTE_NAME%>
+            KeyType: HASH
+        BillingMode: PAY_PER_REQUEST
+        AttributeDefinitions:
+          - AttributeName: <%=CONTAINER_ID_AND_TRIGGER_COMPOSITE_ATTRIBUTE_NAME%>
+            AttributeType: S
+        TimeToLiveSpecification:
+          AttributeName: <%=TIME_TO_LIVE_ATTRIBUTE_NAME%>
+          Enabled: true
```

Comment:
Would it be useful for debugging the issue to keep the table row around? Since the row will be manually added, the table likely won't grow very much and can be manually cleaned up if we ever need to. Also, there likely won't be ID collisions because (IIRC), the lambda IDs are UUIDs.

Reply:
Are we scanning the table regularly, or are we looking up by key? Scans get more expensive as the table grows.

Reply:
@jmkulwik good point - you're right, it shouldn't be too large and yeah lambda IDs are UUIDs. I think it makes sense to remove the TTL for now.

Reply:
@cat5inthecradle we'll only be looking up by key, but frequently (during every Javabuilder session) and writing very infrequently (only when we need to force a container shutdown).